crates/nu-command/src/bytes/at.rs

```diff
@@ -159,3 +146,88 @@ fn action(input: &Value, args: &Arguments, head: Span) -> Value {
         ),
     }
 }
+
+fn handle_byte_stream(
+    args: &Arguments,
+    stream: ByteStream,
+    call: &Call,
+    metadata: Option<nu_protocol::PipelineMetadata>,
+    engine_state: &EngineState,
+) -> Result<PipelineData, ShellError> {
+    let idxs = args.indexes;
+    if idxs.0 >= idxs.1 {
```

Comment:
We should not make this early return.  For example: `0x[01 02 03] | bytes at 1..-1` should returns `0x[02 03]`.

Reply:
Removed - added an example to cover the tests.